plugin-packages/multimedia/src/video/video-component.ts

```
@@ -130,56 +103,37 @@ export class VideoComponent extends BaseRenderComponent {
       transparent = false,
     } = options;
-    let renderer = data.renderer;
 
     this.transparent = transparent;
-    if (!renderer) {
-      renderer = {} as SpriteItemProps['renderer'];
-    }
-    if (video) {
-      this.video = (video as unknown as Asset<HTMLVideoElement>).data;
+    if (video && video.id) {
+      const videoAsset = this.engine.findObject<Asset<HTMLVideoElement>>(video);
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)